ui/raidboss/data/07-dt/trial/zoraal-ja-ex.txt

```diff
@@ -0,0 +1,122 @@
+188.1 "Forward Edge/Backward Edge" #Ability { id: "9972|937F", source: "Zoraal Ja" }
+189.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+194.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+198.4 "Actualize" Ability { id: "9398", source: "Zoraal Ja" }
+210.5 "Projection of Turmoil" Ability { id: "9A88", source: "Zoraal Ja" }
+260.4 "Bitter Whirlwind" Ability { id: "993E", source: "Zoraal Ja" }
+263.7 "Bitter Whirlwind" Ability { id: "9940", source: "Zoraal Ja" }
+266.8 "Bitter Whirlwind" Ability { id: "9940", source: "Zoraal Ja" }
+
+# Tiles phase
+282.1 "Dawn of an Age" Ability { id: "9397", source: "Zoraal Ja" } window 5,5
+299.4 "Drum of Vollok" Ability { id: "938E", source: "Zoraal Ja" }
+300.1 "Drum of Vollok" Ability { id: "938F", source: "Zoraal Ja" }
+311.1 "Vollok" Ability { id: "9392", source: "Zoraal Ja" }
+320.2 "Sync" Ability { id: "9359", source: "Zoraal Ja" }
+329.2 "Chasm of Vollok" Ability { id: "9389", source: "Zoraal Ja" }
+329.3 "Aero III" #Ability { id: "9390", source: "Zoraal Ja" }
+343.6 "Half Full" Ability { id: "939E|9380", source: "Zoraal Ja" } window 5,5
+343.6 "Forward Edge/Backward Edge" #Ability { id: "9972|937F", source: "Zoraal Ja" }
+349.3 "Duty's Edge (cast)" Ability { id: "9374", source: "Zoraal Ja" } window 5,5
+349.7 "Duty's Edge (damage 1)" #Ability { id: "94A7", source: "Zoraal Ja" }
+351.4 "Duty's Edge (damage 2)" #Ability { id: "94A7", source: "Zoraal Ja" }
+353.1 "Duty's Edge (damage 3)" #Ability { id: "94A7", source: "Zoraal Ja" }
+354.9 "Duty's Edge (damage 4)" #Ability { id: "94A7", source: "Zoraal Ja" }
+360.8 "Burning Chains (cast)" Ability { id: "9395", source: "Zoraal Ja" } window 5,5
+367.6 "Burning Chains (damage 1)" #Ability { id: "9396", source: "Zoraal Ja" }
+379.4 "Actualize" Ability { id: "9398", source: "Zoraal Ja" } window 5,5
+
+# Arena phase
+392.8 "Projection of Triumph" Ability { id: "938A", source: "Zoraal Ja" }
+402.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+402.8 "Projection of Triumph" Ability { id: "938A", source: "Zoraal Ja" }
+407.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+412.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+414.7 "Half Circuit" Ability { id: "936[BC]", source: "Zoraal Ja" }
+417.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+422.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+427.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+430.8 "Regicidal Rage" Ability { id: "993C", source: "Zoraal Ja" }
+442.9 "Projection of Turmoil" Ability { id: "9A88", source: "Zoraal Ja" }
+465.4 "Half Full" Ability { id: "939E|9380", source: "Zoraal Ja" }
+474.6 "Half Full" Ability { id: "939E|9380", source: "Zoraal Ja" }
+484.9 "Half Full" Ability { id: "939E|9380", source: "Zoraal Ja" }
```

Comment:
Looks like perhaps these could be numbered?

Reply:
There are other (personal) mechanics that need to be resolved during these, and they're far enough apart that it doesn't make sense to number them.